src/lib/Map.svelte

```diff
@@ -89,13 +111,55 @@
 			// Change the cursor to a pointer when the mouse is over the moments layer.
 			map.on('mouseenter', 'moments-layer', function () {
 				map.getCanvas().style.cursor = 'pointer';
+			
+				openPopup = false;
 			});
-
+			
 			// Change it back to a pointer when it leaves.
 			map.on('mouseleave', 'moments-layer', function () {
 				map.getCanvas().style.cursor = '';
+				openPopup = true;
 			});
 		});
+
+		const app = new SvelteToast({
+		// Set where the toast container should be appended into
+		target: document.body,
+		props: {
+			options: {
+				// Optionally set default options here
+			}
+		}
+	});
+	
+	toast.push(
+			// 'Your story was successfully submitted. It will appear publicly on the map once it has been approved by our moderators.',
+			'<a style="color:var(--color-pink);text-decoration:unset;" href="https://www.patreon.com/queeringthemap" rel="noopener" target="_blank"><span style="color:var(--color-pink);text-decoration:underline;">DONATE</span> <span style="color:var(--color-pink);">to Queering the Map to help us stay online.</span></a> ',
+			{
+				target: 'new',
+				classes: ['toastlinkDonate'],
+				initial: 0,
+				theme: {
+					'--toastBarHeight': 0
+				}
+			}
+		);
+	});
+
+	function handleClick() {
+		document.addEventListener('click', (e) => {
```

Comment:
I'm actually not sure what's happening here. Manually adding event listeners to the DOM like this and then doing a lot of checks to see what was clicked seems un-Svelte-y. And the end-result is "just" clicking the link that was originally clicked? If something complex is happening here, maybe add a comment?